Dockerfile

```diff
@@ -0,0 +1,16 @@
+
+FROM python:3.8-slim-buster as base
+WORKDIR /app
+COPY pyproject.toml .
+RUN pip install poetry
```

Comment:
Minor, but a bit nicer (though more fiddly) to avoid the dependency on pip here, and install poetry as [it suggests](https://python-poetry.org/docs/) with curl.